Packages/Sites/Sfi.Kateheo/Resources/Public/Legacy/Styles/custom.css

```diff
@@ -226,10 +226,24 @@ ul.news-list-view.sfi-kateheo-category, ul.publicaton-list-view.sfi-kateheo-cate
     color: #888;
 }
 
+.news .footer span {
+  display: inherit;
+  padding-right: 0px;
+  margin-right: 0px;
+  border-right: 1px solid #ddd;
+}
+
 span.news-list-date, span.news-list-author{
```

Comment:
You can just write in one selector `span.news-list-date, span.news-list-tags, span.news-list-author`